Release v0.8.0

Issue tracker for the CAA release as part of [CoCo v0.8.0](https://github.com/confidential-containers/confidential-containers/issues/145)

## v0.8.0-alpha.1

- [x] IBM Cloud amd64
- [x] IBM Cloud s390x
- [x] Azure, AKS, SEV-SNP, Ubuntu 22.04
- [x] Libvirt amd64
- [ ] AWS EKS

> @stevenhorsman managing to run the alpha.1 release with AWS

The simple pod failed on AWS (AKS) and remaining tests didn't run because it hit the timeout:
```
=== RUN   TestAwsCreateSimplePod
=== RUN   TestAwsCreateSimplePod/SimplePeerPod_test
    assessment_runner_test.go:190: timed out waiting for the condition
--- FAIL: TestAwsCreateSimplePod (900.53s)
    --- FAIL: TestAwsCreateSimplePod/SimplePeerPod_test (900.53s)
```

I will get more information.

FYI @bpradipt

> > @stevenhorsman managing to run the alpha.1 release with AWS
> 
> The simple pod failed on AWS (AKS) and remaining tests didn't run because it hit the timeout:
> 
> ```
> === RUN   TestAwsCreateSimplePod
> === RUN   TestAwsCreateSimplePod/SimplePeerPod_test
>     assessment_runner_test.go:190: timed out waiting for the condition
> --- FAIL: TestAwsCreateSimplePod (900.53s)
>     --- FAIL: TestAwsCreateSimplePod/SimplePeerPod_test (900.53s)
> ```

Hey Wainer, does your cluster have multiple worker nodes by any chance. I've just realised that my nydus verification tests just runs on the first CAA ds it finds, so wouldn't be reliable on a mutli node cluster. I'm looking to fix it soon, but can back out the test failure in the short-term if it's causing issues?

> > > @stevenhorsman managing to run the alpha.1 release with AWS
> > 
> > 
> > The simple pod failed on AWS (AKS) and remaining tests didn't run because it hit the timeout:
> > ```
> > === RUN   TestAwsCreateSimplePod
> > === RUN   TestAwsCreateSimplePod/SimplePeerPod_test
> >     assessment_runner_test.go:190: timed out waiting for the condition
> > --- FAIL: TestAwsCreateSimplePod (900.53s)
> >     --- FAIL: TestAwsCreateSimplePod/SimplePeerPod_test (900.53s)
> > ```
> 
> @wainersm - Hey Wainer, does your cluster have multiple worker nodes by any chance. I've just realised that my nydus verification tests just runs on the first CAA ds it finds, so wouldn't be reliable on a mutli node cluster. I'm looking to fix it soon, but can back out the test failure in the short-term if it's causing issues?

Hi @stevenhorsman , it is a single-node cluster but good to know tests aren't reliable on a multi-node cluster!

The problem is actually the region the framework deployed the cluster does not support confidential VM, which is now the default since commit e4059a5223bf4a955391b7f87178af9b11809dc2 . I will disable CVM and see if it passes the simple tests.

--- a/install/overlays/aws/kustomization.yaml
+++ b/install/overlays/aws/kustomization.yaml
@@ -7,7 +7,7 @@ bases:
 images:
 - name: cloud-api-adaptor
   newName: quay.io/confidential-containers/cloud-api-adaptor # change image if needed
-  newTag: latest
+  newTag: d4496d008b65c979a4d24767979a77ed1ba21e76
 
 generatorOptions:
   disableNameSuffixHash: true

--- a/install/overlays/azure/kustomization.yaml
+++ b/install/overlays/azure/kustomization.yaml
@@ -7,7 +7,7 @@ bases:
 images:
 - name: cloud-api-adaptor
   newName: quay.io/confidential-containers/cloud-api-adaptor # change image if needed
-  newTag: latest
+  newTag: d4496d008b65c979a4d24767979a77ed1ba21e76
 
 generatorOptions:
   disableNameSuffixHash: true

--- a/install/overlays/ibmcloud-powervs/kustomization.yaml
+++ b/install/overlays/ibmcloud-powervs/kustomization.yaml
@@ -7,7 +7,7 @@ bases:
 images:
 - name: cloud-api-adaptor
   newName: quay.io/confidential-containers/cloud-api-adaptor # change image if needed
-  newTag: latest
+  newTag: d4496d008b65c979a4d24767979a77ed1ba21e76
 
 generatorOptions:
   disableNameSuffixHash: true

--- a/install/overlays/ibmcloud/kustomization.yaml
+++ b/install/overlays/ibmcloud/kustomization.yaml
@@ -7,7 +7,7 @@ bases:
 images:
 - name: cloud-api-adaptor
   newName: quay.io/confidential-containers/cloud-api-adaptor # change image if needed
-  newTag: latest
+  newTag: d4496d008b65c979a4d24767979a77ed1ba21e76
 
 generatorOptions:
   disableNameSuffixHash: true

--- a/install/overlays/libvirt/kustomization.yaml
+++ b/install/overlays/libvirt/kustomization.yaml
@@ -7,7 +7,7 @@ bases:
 images:
 - name: cloud-api-adaptor
   newName: quay.io/confidential-containers/cloud-api-adaptor # change image if needed
-  newTag: latest
+  newTag: dev-d4496d008b65c979a4d24767979a77ed1ba21e76
 
 generatorOptions:
   disableNameSuffixHash: true

--- a/install/overlays/vsphere/kustomization.yaml
+++ b/install/overlays/vsphere/kustomization.yaml
@@ -7,7 +7,7 @@ bases:
 images:
 - name: cloud-api-adaptor
   newName: quay.io/confidential-containers/cloud-api-adaptor # change image if needed
-  newTag: latest
+  newTag: d4496d008b65c979a4d24767979a77ed1ba21e76
 
 generatorOptions:
   disableNameSuffixHash: true

